apps/auth/config.py

```
@@ -27,3 +27,37 @@
+        "\033[31m[ERROR]\033[0m \033[107mISSUER not set, please set it in the .env file\033[0m"
+    )
+ISSUER = os.getenv("ISSUER")
+if not os.getenv("AUTH_KEY"):
+    raise ValueError(
+        "\033[31m[ERROR]\033[0m \033[107mAUTH_KEY not set, please set it in the .env file\033[0m"
+    )
+AUTH_KEY = os.getenv("AUTH_KEY")
+if not os.getenv("ISSECURE"):
+    print("\033[33m[WARN]\033[0m ISSECURE not set, using default value of False")
+try:
+    ISSECURE = bool(os.getenv("ISSECURE", False))
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)